pyresample/bucket/__init__.py

```diff
@@ -169,12 +176,12 @@ def get_sum(self, data, mask_all_nan=False):
         sums, _ = da.histogram(self.idxs, bins=out_size, range=(0, out_size),
                                weights=weights, density=False)
 
-        if mask_all_nan:
+        # TODO remove following lines in favour of weights = data when dask histogram bug (issue #6935) is fixed
```

Comment:
Can you make a separate method for the lines that would have to be removed when the bug is fixed?

Reply:
Done in a new commit, is it ok like this?

Reply:
Thanks, looks good